ui/src/containers/VolumeInformation.js

```diff
@@ -127,13 +155,14 @@ const VolumeInformation = props => {
         <InformationSpan>
           <InformationLabel>{intl.messages.path}</InformationLabel>
           <InformationValue>
-            {pV && pV.spec && pV.spec.local && pV.spec.local.path}
+            {volume?.spec?.rawBlockDevice?.devicePath ||
```

Comment:
minor: IMHO, this change should be in a dedicated commit: this THE change that fix the bug.

The banner is a great UX improvement, but not (directly) related to the bug reported so it would make sense to have one commit for the banner and one commit for the bugfix.